### PR TITLE
[#138] update readme

### LINKED ISSUE #138: Update README.md
- add maven dependency badge (ref https://shields.io/)
- re-order badge
- update description and links

### PR DESCRIPTION
**Pull request by ztkmkoo**
- add maven dependency badge
- re-order badge
- update guides link

I suggest that view README.md as file (not raw file)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,25 @@
...
-https://github.com/ztkmkoo/dss/blob/develop/CODE_OF_CONDUCT.md
-
-## TUTORIAL
+## GUIDE
 
-https://github.com/ztkmkoo/dss/wiki/Project-Tuturial(KR)
\ No newline at end of file
+- [Project Structure](https://github.com/ztkmkoo/dss/wiki/Project-Structure)
+- [Code Convention](https://github.com/ztkmkoo/dss/wiki/Code-Style)
+- [Tutorial](https://github.com/ztkmkoo/dss/wiki/Project-Tuturial)
```
Comment (Doyuni) [inline README.md]: It seems that there is no project-tutorial page, is right?

Comment (RulLu16) [reply README.md]: I agree. need to change it to https://github.com/ztkmkoo/dss/wiki/Project-Tutorial(KR), I guess. 
Or are you going to make another tutorial-page? I just wondering.

Comment (ztkmkoo) [reply README.md]: Yap.. i see,
i create https://github.com/ztkmkoo/dss/wiki/Dss-REST-Tuturial this one.
Thanks~

Comment (ztkmkoo) [reply README.md]:
``` suggestion
- [Tutorial](https://github.com/ztkmkoo/dss/wiki/Dss-REST-Tuturial)
```

Comment (ztkmkoo) [inline README.md]:
``` suggestion
[Tutorial](https://github.com/ztkmkoo/dss/wiki/Project-Tutorial)
```

File: README.md [hunk 2]
```diff
@@ -1,33 +1,25 @@
...
-https://github.com/ztkmkoo/dss/blob/develop/CODE_OF_CONDUCT.md
-
-## TUTORIAL
+## GUIDE
 
-https://github.com/ztkmkoo/dss/wiki/Project-Tuturial(KR)
\ No newline at end of file
+- [Project Structure](https://github.com/ztkmkoo/dss/wiki/Project-Structure)
+- [Code Convention](https://github.com/ztkmkoo/dss/wiki/Code-Style)
+- [Tutorial](https://github.com/ztkmkoo/dss/wiki/Dss-REST-Tuturial)
```
Comment (ztkmkoo) [inline README.md]:
``` suggestion
- [Tutorial](https://github.com/ztkmkoo/dss/wiki/Dss-REST-Tutorial)
```

**Review by Alencion — APPROVED** [review]

**Review by RulLu16 — APPROVED** [review]

**Review by JuHyun419 — APPROVED** [review]

**Pull request merged**